pom.xml

```diff
@@ -407,23 +408,7 @@
 			<version>${spark.version}</version>
 			<scope>test</scope>
 		</dependency>
-		<!--		<dependency>-->
-		<!--			<groupId>org.apache.spark</groupId>-->
-		<!--			<artifactId>spark-streaming_${scala.minor.version}</artifactId>-->
-		<!--			<version>${spark.version}</version>-->
-		<!--		</dependency>-->
 
-		<!--		<dependency>-->
-		<!--			<groupId>org.apache.spark</groupId>-->
-		<!--			<artifactId>spark-streaming-flume_${scala.minor.version}</artifactId>-->
-		<!--			<version>${spark.version}</version>-->
-		<!--		</dependency>-->
-
-		<!--		<dependency>-->
-		<!--			<groupId>org.apache.spark</groupId>-->
-		<!--			<artifactId>spark-yarn_${scala.minor.version}</artifactId>-->
```

Comment:
Will be needed on a yarn cluster, but not in smartdatalake library, that's true.
@pgruetter: please note for migration guide...

Reply:
Done, will check again when migrating the first project.